custom-actions/media-panel/create_dated_folder.py

```diff
@@ -0,0 +1,34 @@
+#Creates a Dated Folder
+def get_media_panel_custom_ui_actions():
+
+    def scope_libraries(selection):
+        import flame
+        for item in selection:
+            if isinstance(item, (flame.PyLibrary)):
```

Comment:
No need for () pair around flame.PyLibrary, isinstance() work on either a single class name or a tuple of class name. In this case, since there is only one to check against, having () is not useful.